src/app/zap-templates/zcl/data-model/chip/refrigerator-alarm.xml

```diff
@@ -28,14 +28,12 @@ limitations under the License.
     <define>REFRIGERATOR_ALARM</define>
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
-    <attribute side="server" code="0x0000" define="MASK" type="AlarmMap" default="0" writable="true"  optional="false">Mask</attribute>
-    <attribute side="server" code="0x0001" define="LATCH" type="AlarmMap" default="0" writable="false" optional="false">Latch</attribute>
-    <attribute side="server" code="0x0002" define="STATE" type="AlarmMap" default="0" writable="false" optional="false">State</attribute>
+    <attribute side="server" code="0x0000" define="MASK" type="AlarmMap" default="0" writable="false" optional="false">Mask</attribute>
+    <attribute side="server" code="0x0001" define="STATE" type="AlarmMap" default="0" writable="false" optional="false">State</attribute>
 
-    <command source="client" code="0x00" name="Reset" optional="false">
-        <description>Reset alarm</description>
-        <arg name="Alarms" type="AlarmMap" optional="false"/>
-        <arg name="Mask" type="AlarmMap" optional="true"/>
+    <command source="client" code="0x00" name="ModifyEnabledAlarms" optional="true">
```

Comment:
I guess this matches the spec in terms of being optional, but since per spec this command _always fails_ for this cluster, why do we allow it to start with?  Spec issue, please?